src/main/java/racingcar/model/Register.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Register {
+    public ArrayList<Car> carList;
```

Comment:
캡슐화에 대해서 알아보시면 좋을 것 같아요
```suggestion
public class Register {
    private List<Car> cars;
```